packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -192,9 +192,12 @@ model = GLiNER.from_pretrained("${model.id}")`,
 ];
 
 export const keras = (model: ModelData): string[] => [
-	`from huggingface_hub import from_pretrained_keras
+	`# Available backend options are: "jax", "tensorflow", "torch".
+os.environ["KERAS_BACKEND"] = "tensorflow"
```

Comment:
(unrelated to this PR) is it automatic if one backend is installed in the env?

Reply:
I'd say yes but not sure. They insisted in having this line in the code snippet in model cards so I kept it

Reply:
put the import os between the comment and the os.environ no?

Reply:
done in https://github.com/huggingface/huggingface.js/pull/774/commits/55947f7cd259c3c465933862c78cff9fbb2520a6